native/SampleApps/RestAPIExplorer/RestAPIExplorer/Helpers/UIFont+helper.swift

```diff
@@ -1,5 +1,10 @@
 /*
- Copyright (c) 2011-present, salesforce.com, inc. All rights reserved.
+ UIFont+helper.swift
```

Comment:
I don't think we need this file. Let's get rid of the `Salesforce` fonts and use the default one.

Reply:
@bhariharan I don't see any harm to  keeping it around since the font theme can be controlled in one place. Ive replaced the impl to use system fonts.

Reply:
Sure - no harm, but I don't know if it serves any purpose in this app. I'm okay either way. I'll leave it up to you.